api/v1alpha3/machinedeployment_types.go

```diff
@@ -147,6 +147,13 @@ type MachineRollingUpdateDeployment struct {
 	// at any time during the update is at most 130% of desired machines.
 	// +optional
 	MaxSurge *intstr.IntOrString `json:"maxSurge,omitempty"`
+
+	// DeletePolicy defines the policy used by the MachineSet to identify nodes to delete when downscaling.
```

Comment:
s/MachineSet/MachineDeployment?

Reply:
I meant to say, it's used by the underlying machineset. Should I just leave it as `// DeletePolicy defines the policy used identify the nodes to delete when downscaling.` to avoid confusion?